### DISCUSSION TITLE
스터디 - 8주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#72] [General]
> 📢기존 팀원이 했던 거 리팩토링을 하고 싶은 경우 제목 옆에 추가 또는 리팩토링 이라고 적어주세요.
🏥 겨울 입니다. 모두 몸 관리 잘하시고, 다들 부담 갖지말고 스터디 잘 진행주세요 화이팅!

## 시간표
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접, 모의면접은 상대방이 공부한 것으로 기준으로 2개 정도 질문하고 피드백 해준다.
21:35 ~ 21:50 모의 면접을 했던 상대방이 아닌 다른 사람에게도 질문하고 싶은 걸 한다.
21:50 ~ 22:00 해당 주차에 회고록 작성, 프론트엔드 질문 정리 질문 했던 댓글 남기기.

## 송현
-  React - props.children

## 영생
- TypeScript - 제네릭(Generic)

## 동기
- JavaScript - event handler

## 두현(9주차 스터디 목록)
- JavaScript - Webpack(리팩토링)
- JavaScript - Prototype

## 수민
- React - Recociliation (재조정)

## 남은 
- JavaScript - Closure 활용사례(커링함수, 쓰로틀링 등)

**Comment by Doodream** [comment]
두현
JavaScript - Webpack
JavaScript - Prototype

**Reply by Doodream** [reply]
다음주로 연기하겠습니다.

**Comment by ki7348** [comment]
동기
JavaScript - 이벤트 핸들러

**Comment by NAMU1105** [comment]
JavaScript - Closure 활용사례(커링함수, 쓰로틀링 등)

**Comment by whdudtod1273** [comment]
TypeScript - 제네릭(Generic)

**Comment by ttumzzi** [comment]
React - Recociliation (재조정)
